feature-charity/src/main/java/com/hedvig/android/feature/charity/ui/CharityBottomSheet.kt

```diff
@@ -0,0 +1,71 @@
+package com.hedvig.android.feature.charity.ui
+
+import androidx.activity.compose.BackHandler
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.WindowInsets
+import androidx.compose.foundation.layout.WindowInsetsSides
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.only
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.safeDrawing
+import androidx.compose.foundation.layout.windowInsetsPadding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.hedvig.android.core.designsystem.component.BottomSheetHandle
+import hedvig.resources.R
+
+@Composable
+internal fun CharityBottomSheet(
+  isShowing: Boolean,
+  closeSheet: () -> Unit,
+) {
+  BackHandler(
+    enabled = isShowing,
+    onBack = { closeSheet() },
```

Comment:
```suggestion
    onBack = ::closeSheet,
```

(on condition that `onBack: () -> Unit`, of course)

Reply:
Yup totally agree with this one!
Had to use the `onback = closeSheet` syntax since `closeSheet` is itself of type `() -> Unit` and not a function which we would need to do `::` on